tests/windows_test.go

```diff
@@ -380,3 +454,28 @@ func runCommandAndExpectOutput(virtClient kubecli.KubevirtClient, winrmcliPod *k
 		return output
 	}, time.Minute*1, time.Second*10).Should(MatchRegexp(expectedOutputRegex))
 }
+
+func isTSCFrequencyExposed(virtClient kubecli.KubevirtClient) bool {
+	nodeList, err := virtClient.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
+	Expect(err).ToNot(HaveOccurred())
+
+	// Usually TSC frequency is either exposed on all nodes or none of them. Therefore, if one node is found to not
```

Comment:
While this might be true I don't see why we need to require all nodes to have TSC.